lib/validators/liveValidator.ts

```diff
@@ -825,7 +825,20 @@ export class LiveValidator {
     }
   }
 
-  private async getSwaggerInitializer(swaggerPath: string): Promise<void> {
+  public async refreshResourceProviderCache(resourceProvider: string): Promise<void> {
+    const specsPaths = (await this.getSwaggerPaths()).filter(
```

Comment:
Leverage options.swaggerPathsPattern which will be used to match the swagger to load.